README.md

```diff
@@ -178,12 +178,38 @@ Notes:
 * The AutoScaling groups it runs against can be in any region, since all regions
   are processed at runtime.
 
+#### Installation options ####
+
+The CloudFormation stack takes a number of parameters, which allow you to
```

Comment:
I think it should be `which allows` as it refers to `a number of parameters`; but could be wrong.